lib/cli/commands/walletwithdraw.ts

```diff
@@ -1,16 +1,16 @@
 import { Arguments, Argv } from 'yargs';
 import { WithdrawRequest } from '../../proto/xudrpc_pb';
 import { callback, loadXudClient } from '../command';
-import { coinsToSats } from '../utils';
+import { argChecks, coinsToSats } from '../utils';
 
-export const command = 'walletwithdraw [amount] [currency] [destination] [fee]';
+export const command = 'walletwithdraw <amount> <currency> <destination> [fee]';
```

Comment:
I think we want to leave destination as optional as it's not required for connext.

Reply:
Gotcha